libres/lib/enkf/obs_data.cpp

```diff
@@ -402,8 +401,9 @@ matrix_type *obs_data_allocE(const obs_data_type *obs_data, rng_type *rng,
         for (iobs_active = 0; iobs_active < active_obs_size; iobs_active++)
             pert_mean[iobs_active] += matrix_iget(E, iobs_active, iens);
 
-    for (iobs_active = 0; iobs_active < active_obs_size; iobs_active++)
+    for (iobs_active = 0; iobs_active < active_obs_size; iobs_active++) {
```

Comment:
Why { and } ?

Reply:
Forgot to remove it after some debugging. Thanks!